### PR TITLE
Regenerate HTML before updating to fix the old style issues

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
This PR aims to update HTML result of the current HEAD to fix style issues.
```
PRODUCTION=1 jekyll build
```

**developer-tools.html**
developer-tools

### CODE REVIEW

**Comment by srowen** [comment]
This is with the very latest jekyll? yes that's a fine idea.

**Comment by dongjoon-hyun** [comment]
Yes. Right. It seems to have some bug fixes.

File: site/downloads.html [hunk 1]
```diff
@@ -207,7 +207,7 @@ <h5>Latest News</h5>
 });
 </script>
 
-<h2 id="download-apache-sparktrade">Download Apache Spark&#8482;</h2>
+<h2 id="download-apache-spark">Download Apache Spark&#8482;</h2>
```
Comment (dongjoon-hyun) [inline site/downloads.html:210]: This one seems due to a rule change. I checked this id is uniquely used once here.

**Comment by dongjoon-hyun** [comment]
Thank you for review, @srowen . I'll merge this and move forward.

**Pull request merged**